ubtue_Peeters.js

```diff
@@ -197,7 +169,6 @@ function scrape(doc, url) {
 	item.url = url;
 
 	parseAbstract(doc, item);
-	if (item.abstractNote.match(/not\s?available+|^editorial$|^Obituary$/i)) delete item.abstractNote;
```

Comment:
Was this intentionally removed?

Reply:
Yes, abs.length > 100 covers these cases, too.